po/exiv2.pot

```diff
@@ -13401,7 +13401,7 @@ msgstr ""
 
 #: src/properties.cpp:277
 msgid ""
-"References to resources that were incorporated, byinclusion or reference, "
+"References to resources that were incorporated, by inclusion or reference, "
```

Comment:
I've mostly ignored the `.pot` file (and probably should have ignored it entirely).